Levels/MGZ/Debug/Main.asm

```diff
@@ -10,7 +10,7 @@
 	dbglistobj Obj_Spring,			Map_Spring,			 $40,  $A, make_art_tile($478,0,0)
 	dbglistobj Obj_Spikes,			Map_Spikes,			   0,   0, make_art_tile($49C,0,0)
 	dbglistobj Obj_Spikes,			Map_Spikes,			 $40,   4, make_art_tile($494,0,0)
-	dbglistobj Obj_MGZMiniBoss,		Map_MGZMiniBoss,		   0,   0, make_art_tile($54F,1,0)
+	dbglistobj Obj_Tunnelbot,		Map_MGZMiniBoss,		   0,   0, make_art_tile($54F,1,0)
```

Comment:
Shouldn't the mappings be renamed to match as well?

Reply:
Hmm, the mappings are shared between the miniboss object and the enemy... The problem here was that the enemy was erroneously labeled as the miniboss. I think that given where the data appears in the S3 ROM, the name of the mappings should be definitely be Miniboss, but I've gone back and fixed the weird casing which I missed the first time.